.github/workflows/test.yml

```diff
@@ -31,13 +37,16 @@ jobs:
           cd test/${{ matrix.package-manager }}
           # Install dependencies
           ni
-          # run tests
-          ni test
-          # run scripts
-          ni run main
-          # exec envinfo
-          ni exec envinfo
-          # add new dependencies
-          ni add lodash
-          # remove dependencies
-          ni remove lodash
+          # Test package management commands
+          if [ "${{ matrix.package-manager }}" = "deno" ]; then
+            # Run tests
+            ni test
+            # Run main script
+            ni run main
```

Comment:
https://github.com/azu/ni.zsh?tab=readme-ov-file#command-table
deno already support `add` and `remove`